app/assets/stylesheets/_fonts.css

```diff
@@ -1,4 +1,4 @@
 @font-face {
   font-family: "Roboto-Medium";
-  src: url(font-path("Roboto-Medium.ttf"));
+  src: url("Roboto-Medium.ttf");
```

Comment:
no es necesario cambiar en este PR, pero creo q deberiamos re-evaluar si es necesario tener `Roboto-Medium `, pareceria que lo usamos solo para el boton de sign in con google